pcsx2/gui/Panels/SpeedhacksPanel.cpp

```diff
@@ -309,12 +310,14 @@ void Panels::SpeedHacksPanel::ApplyConfigToGui( AppConfig& configToApply, int fl
 	SetVUcycleSliderMsg();
 
 	m_check_vuFlagHack->SetValue(opts.vuFlagHack);
-	if( !(flags & AppConfig::APPLY_FLAG_FROM_PRESET) )
-		m_check_vuThread	->SetValue(opts.vuThread);
 	m_check_intc->SetValue(opts.IntcStat);
 	m_check_waitloop->SetValue(opts.WaitLoop);
 	m_check_fastCDVD->SetValue(opts.fastCDVD);
 
+	bool preset_request = flags & AppConfig::APPLY_FLAG_FROM_PRESET;
+	if (!preset_request || (preset_request && configToApply.PresetIndex == 0))
```

Comment:
Could be simplified to:
`if (!preset_request || configToApply.PresetIndex == 0)`

Not sure you need the `preset_request` variable anymore. Probably should be made const if you do keep it.